nmrfx-analyst-gui/src/main/java/org/nmrfx/processor/gui/ProcessorController.java

```diff
@@ -231,40 +228,48 @@ public String toString() {
     private AtomicBoolean needToFireEvent = new AtomicBoolean(false);
     private AtomicReference<Dataset> saveObject = new AtomicReference<>();
     ScheduledFuture futureUpdate = null;
+    private boolean viewIfPossible = true;
 
 
-    public static ProcessorController create(FXMLController fxmlController, StackPane processorPane, PolyChart chart) {
-        Fxml.Builder builder = Fxml.load(ProcessorController.class, "ProcessorScene.fxml")
-                .withParent(processorPane);
+    public static ProcessorController create(FXMLController fxmlController, NmrControlRightSidePane nmrControlRightSidePane, PolyChart chart) {
+        Fxml.Builder builder = Fxml.load(ProcessorController.class, "ProcessorScene.fxml");
         ProcessorController controller = builder.getController();
 
         controller.chart = chart;
         chart.setProcessorController(controller);
         controller.chartProcessor.setChart(chart);
         controller.chartProcessor.setFxmlController(fxmlController);
-        controller.processorPane = processorPane;
-        controller.pane = builder.getNode();
```

Comment:
controller.pane is just the top level Pane in the ProcessorScene.fxml that had an fx:id ("mainBox") already, so it doesn't make sense to call it a different name, so switched to setting it using the @FXML tag